packages/suite/src/components/suite/SuiteLayout/index.tsx

```diff
@@ -141,6 +142,27 @@ const BodyWide = ({ url, menu, appMenu, children, guideOpen }: BodyProps) => (
     </Body>
 );
 
+/**
+ * Displays menu and body in two columns when guide is closed.
+ * Displays menu inlined in body when guide is open.
+ */
+const BodyNormal = ({ url, menu, appMenu, children, guideOpen }: BodyProps) => (
+    <Body>
+        <Columns>
+            {!guideOpen && menu && <MenuSecondary>{menu}</MenuSecondary>}
+            <ScrollAppWrapper url={url}>
+                {guideOpen && menu}
+                {appMenu}
+                <DefaultPaddings>
+                    <MaxWidthWrapper>{children}</MaxWidthWrapper>
+                </DefaultPaddings>
+            </ScrollAppWrapper>
+            {guideOpen && <StyledGuidePanel />}
+        </Columns>
+    </Body>
+);
+
```

Comment:
Here (and below at lines 217-226) I traded a small code duplicity for more straight forward code and easier to read doc-comments. However, I'm open to any advice on how to make this more concise.

Reply:
I don't like the duplicity. However, it is true, that thanks to this, the magic here is more understandable.

I have an idea to merge `BodyWide` and `BodyNormal` into one function. Something like this (I am not 100% happy with it, needs some polishing):

```
// also you can exchange layoutSize for isXLargeLayout directly
const BodyDesktop = ({ url, menu, appMenu, children, guideOpen, layoutSize }: BodyProps) => {
    // your previous comments
    const isXLargeLayout = layoutSize === 'XLARGE';

    return (
        <Body>
            <Columns>
                {menu && (isXLargeLayout || (!isXLargeLayout && !guideOpen)) && (
                    <MenuSecondary>{menu}</MenuSecondary>
                )}
                <ScrollAppWrapper url={url}>
                    {!isXLargeLayout && guideOpen && menu}
                    {appMenu}
                    <DefaultPaddings>
                        <MaxWidthWrapper>{children}</MaxWidthWrapper>
                    </DefaultPaddings>
                </ScrollAppWrapper>
                {guideOpen && <StyledGuidePanel />}
            </Columns>
        </Body>
    );
};
```

Reply:
EDIT:
I do not want to delete the previous comment. But I think that you can merge it to 1 function and somehow use the `isInline` variable mentioned in the last comment.

Reply:
Kinda did here b63cf3eaad38f7c22987ad9c123a35f0607c15f1

If you think that's better, please resolve this. .)

Reply:
I did not test it, but it looks definitely better 👍